examples/batch_loader.c

```diff
@@ -20,7 +20,7 @@
  * Read URLs from stdin and download into results/domain/.
  *
  */
-
+#include "config.h"
 #include <stdio.h>
```

Comment:
Option `-FI./config.h` could also be used.

Reply:
Will do, will add to the examples/Makefile.am